src/main/java/com/techsophy/tsf/account/controller/InternalUserFormDataController.java

```diff
@@ -0,0 +1,16 @@
+package com.techsophy.tsf.account.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.techsophy.tsf.account.dto.InternalUserFormDataSchema;
+import com.techsophy.tsf.account.model.ApiResponse;
+import org.springframework.http.HttpHeaders;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import static com.techsophy.tsf.account.constants.AccountConstants.VERSION_V1;
+
+@RequestMapping("/internal"+VERSION_V1+"/users/create/")
```

Comment:
Lets not create a controller for each internal API, lets keep this as a proxy only

Reply:
only one internal api is there for now

Reply:
does not matter we need to keep the correct structure